src/wpool_process.erl

```diff
@@ -55,12 +55,12 @@ call(Process, Call, Timeout) ->
 %% @equiv gen_server:cast(Process, {cast, Cast})
 -spec cast(wpool:name() | pid(), term()) -> ok.
 cast(Process, Cast) ->
-    gen_server:cast(Process, {cast, Cast}).
+    gen_server:cast(Process, Cast).
 
-%% @equiv gen_server:cast(Process, {call, From, Call})
--spec cast_call(wpool:name() | pid(), from(), term()) -> ok.
-cast_call(Process, From, Call) ->
-    gen_server:cast(Process, {call, From, Call}).
```

Comment:
WOW! I didn't realize we could already remove this super-annoying function. Amazing!